src/utils/utils.py

```diff
@@ -46,6 +46,21 @@ def wrap_singleton_string(item: Union[Sequence, str]):
         return [item]
     return item
 
+def get_tag_list(tag_vocab):
+    '''
+    retrieve tag strings from the tag vocab object
+    Args:t
+        tag_vocab: the vocab that contains all tags 
+    Returns:
+        tag_list: a list of "coarse__fine" tag strings
+    '''
+    # get dictionary from allennlp vocab, neglecting @@unknown@@ and @@padding@@
+    tid2tag_dict = {key-2: tag \
+        for key, tag in tag_vocab.get_index_to_token_vocabulary().items() \
+            if key - 2 >= 0}
+    tag_list = [tid2tag_dict[tid].replace(':', '_').replace(', ', '_').replace(' ', '_').replace('+', '_') for tid in range(len(tid2tag_dict))]
```

Comment:
Why is this needed? IIRC if the vocab ends in `_labels`, then AllenNLP won't add the unk and padding tokens. See https://github.com/jsalt18-sentence-repl/jiant/blob/master/src/preprocess.py#L529

Also, use a regex instead of a chain of `.replace` calls?

Reply:
This is for backwards-compatibility with the original implementation of subset scorers. Shouldn't be too bad to just change the tasks currently using this though.